src/main/resources/assets/immersivepetroleum/manual/en_us/napalm.txt

```diff
@@ -1,3 +0,0 @@
-Napalm
-Beware the River God
-Napalm is an extremely flammable substance that can be created in the <link;mixer;§n§oMixer§r> using <link;immersivepetroleum:distillationtower;§n§oGasoline§r> and three measures of Aluminium Grit. It has little use other than wanton destruction; when Napalm is lit, fire will spread almost immediately over its surface, allowing for rapid deforestation. In addition, it can serve as a very effective fuel for the <link;chemthrower;§n§oChemthrower§r>.
```

Comment:
what, why is i gone, what happend? lol

Reply:
It got merged into a bigger entry! See "Petroleum Products", labeled `fluids` in the text? It's in there now!